test/e2e/shoot/create_migrate_and_delete.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shoot
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	. "github.com/gardener/gardener/test/framework"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultMrExcludeList              = ""
+	defaultResourcesWithGeneratedName = ""
+)
+
+var _ = Describe("Shoot Tests", Label("Shoot"), func() {
```

Comment:
How does this test work? I don't see any second local seed being created to which the shoot could be migrated to?

Reply:
It does not work at the moment. It is still a work in progress.

Reply:
Why is the PR then marked as ready for review?
There should be a script that prow calls that includes `make kind-up`, `make gardener-up` as well as the setup for the second seed. Ultimately, it calls `make test-e2e-local-migration`.
You can call the script `hack/ci-e2e-kind-migration.sh`.